src/Manager.hh

```diff
@@ -20,7 +20,7 @@
 #include <memory>
 #include <string>
 
-// #include <ignition/common/SuppressWarning.hh>
+#include <ignition/utils/SuppressWarning.hh>
```

Comment:
You need to link against `ignition-utils${IGN_UTILS_VER}::core` for this to work. You'd add that in https://github.com/gazebosim/gz-launch/blob/e2a18dde4607db5c51c5fe5efe30aec24198a744/src/CMakeLists.txt#L23

Reply:
Just added it with 7ba196f. But I think the cmd code copies back the Manager.hh source into the cmd folder and then the library is not linked there. 

Should I link it there: https://github.com/gazebosim/gz-launch/blob/e2a18dde4607db5c51c5fe5efe30aec24198a744/src/cmd/CMakeLists.txt#L12 ?

Reply:
Ah, we'll need to link `ignition-utils` there as well then.